src/Microsoft.NET.Test.Sdk.targets

```diff
@@ -23,6 +23,12 @@
     <GeneratedProgramFile Condition="'$(GeneratedProgramFile)' ==''">$(IntermediateOutputPath)$(MSBuildProjectName).Program$(DefaultLanguageSourceExtension)</GeneratedProgramFile>
     <GenerateProgramFile Condition="'$(GenerateProgramFile)' == ''">true</GenerateProgramFile>
   </PropertyGroup>
+
```

Comment:
Please add the bug reference (https://github.com/dotnet/roslyn-project-system/issues/268) here. It will tell us why did we moved OutputType from props file target file. In future once the bug will get fixed we will revert it back.